GPU/GLES/StateMappingGLES.cpp

```diff
@@ -302,6 +302,7 @@ void DrawEngineGLES::ApplyDrawState(int prim) {
 void DrawEngineGLES::ApplyDrawStateLate(bool setStencilValue, int stencilValue) {
 	if (setStencilValue) {
 		render_->SetStencilFunc(GL_TRUE, GL_ALWAYS, stencilValue, 255);
+		render_->SetStencilOp(0xFF, GL_REPLACE, GL_REPLACE, GL_REPLACE);
```

Comment:
This is actually partially wrong.  When in clear mode, the stencil write mask is still applied (previously set by `renderManager->SetStencilOp(stencilState.writeMask, GL_REPLACE, GL_REPLACE, GL_REPLACE);`)  Now it's no longer applied.

It'd be reasonable to apply it here though, not saying it doesn't make sense to do these together.

-[Unknown]

Reply:
darn, yeah you're right - should have figured that this will change behavior, of course.

I'll fix it up.